ssl/quic/quic_obj.c

```diff
@@ -87,3 +88,38 @@ SSL_CONNECTION *ossl_quic_obj_get0_handshake_layer(QUIC_OBJ *obj)
 
     return SSL_CONNECTION_FROM_SSL_ONLY(((QUIC_CONNECTION *)obj)->tls);
 }
+
+/* (Returns a cached result.) */
+int ossl_quic_obj_can_support_blocking(const QUIC_OBJ *obj)
+{
+    QUIC_REACTOR *rtor = ossl_quic_obj_get0_reactor(obj);
+
+    return ossl_quic_reactor_can_poll_r(rtor)
+        || ossl_quic_reactor_can_poll_w(rtor);
+}
+
+int ossl_quic_obj_desires_blocking(const QUIC_OBJ *obj)
+{
+    unsigned int req_blocking_mode;
+
```

Comment:
check for obj == NULL? just asking because at lline 85 we do such check. also assert at line 84 might deserve `obj != NULL` test too.